notebooks/tutorial_ivf_pq.ipynb

```diff
@@ -0,0 +1,1226 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    # Sync the GPU to make sure we've got the timing right
</pre><p>I would propose maybe something more user friendly here like "This function is asychronous so we need to explicitly synchronize the GPU before we can measure the execution time". </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/raft/pull/1544/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>